Client/Frontend/PasswordManagement/BreachAlertsDetailView.swift

```diff
@@ -191,36 +185,56 @@ class BreachAlertsDetailView: UIView, ThemeApplicable {
+                titleLabel.leadingAnchor.constraint(equalTo: titleStack.leadingAnchor, constant: UX.titleIconSize),
+                learnMoreButton.leadingAnchor.constraint(equalTo: titleStack.leadingAnchor, constant: UX.titleIconSize)
+            ])
         } else {
             self.titleStack.axis = .horizontal
             self.titleStack.alignment = .leading
-            self.titleLabel.snp.makeConstraints { make in
-                make.centerY.equalToSuperview()
-            }
-            self.learnMoreButton.snp.makeConstraints { make in
-                make.centerY.equalToSuperview()
-            }
+
+            NSLayoutConstraint.activate([
+                titleLabel.centerYAnchor.constraint(equalTo: titleStack.centerYAnchor),
+                learnMoreButton.centerYAnchor.constraint(equalTo: titleStack.centerYAnchor)
+            ])
         }
     }
 
     // MARK: - ThemeApplicable
     func applyTheme(theme: Theme) {
-        layer.shadowColor = theme.colors.shadowDefault.cgColor
-        backgroundColor = theme.colors.layer2
-        titleIcon.tintColor = theme.colors.iconWarning
-        titleLabel.textColor = theme.colors.textWarning
-        breachDateLabel.textColor = theme.colors.textWarning
-        descriptionLabel.textColor = theme.colors.textWarning
-        goToButton.attributedText = getAttributedText(for: breachLink,
-                                                      with: theme.colors.textWarning)
-        learnMoreButton.setAttributedTitle(getAttributedText(for: .BreachAlertsLearnMore,
-                                                             with: theme.colors.textWarning),
-                                           for: .normal)
+        let colors = theme.colors
+        layer.shadowColor = colors.shadowDefault.cgColor
+        backgroundColor = colors.layer2
+        titleIcon.tintColor = colors.iconWarning
+        titleLabel.textColor = colors.textWarning
+        breachDateLabel.textColor = colors.textWarning
+        descriptionLabel.textColor = colors.textWarning
+        goToButton.attributedText = getAttributedText(
+            for: breachLink,
+            with: colors.textWarning
+        )
+        learnMoreButton.setAttributedTitle(
+            getAttributedText(
+                for: .BreachAlertsLearnMore,
+                with: colors.textWarning
+            ),
+            for: .normal
+        )
+    }
+}
+
+// MARK: - Exposed Properties
+// This extension provides controlled external access to specific private properties of BreachAlertsDetailView.
+
+extension BreachAlertsDetailView {
+    /// Exposes the private `learnMoreButton` property for controlled access from outside the class.
+    public var viewBreachDetailsButton: UIButton {
+        return learnMoreButton
+    }
+
+    /// Exposes the private `goToButton` property for controlled access from outside the class.
+    public var visitBreachSource: UILabel {
+        return goToButton
```

Comment:
I don't think we need to add a wrapper to expose these properties, in fact I don't think we need to expose them at all.